ledger-core/rms/proto_vnvn_validation_vcachedmemoryrequest.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package rms
+
+import "github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
+
+func (m *VCachedMemoryRequest) Validate(currentPulse PulseNumber) error {
+	object := m.Object.GetValue()
+	_, err := validSelfScopedGlobalWithPulseBeforeOrEq(object, currentPulse, "Object")
+	if err != nil {
+		return err
+	}
+
+	if m.Object.IsEmpty() {
```

Comment:
I suppose, `validSelfScopedGlobalWithPulseBeforeOrEq` will fail if object is empty.